assets/dev/scss/helpers/mixins.scss

```diff
@@ -238,4 +238,16 @@
 		border-radius: $border-radius;
 	}
 }
+
+@mixin button-focus() {
```

Comment:
@rami-elementor 
I thinking of creating a general mixin for button focus styling. 
What do you think about that?

Reply:
please explain why not use the existing `@mixin button-focus()` ?

Reply:
I don't think there is an existing mixin.
I created this mixin.